### PR TITLE
[lib/tofdb] Add tofdb tests

### PR DESCRIPTION
**Pull request by szymon-czapracki**

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
@ncasaril FYI

File: lib/tofdb/selftest/src/testcases/tofdb_set_tof_test.c [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+     *
+     */
+    rc = tofdb_set_tof(0x1, 200);
+
+    TEST_ASSERT(rc == OS_OK);
+
+    rc = tofdb_set_tof(0x1, 250);
+
+    TEST_ASSERT(rc == OS_OK);
+
```
Comment (ncasaril) [inline lib/tofdb/selftest/src/testcases/tofdb_set_tof_test.c:114]: You need to get the tof for 0x1 to see that it actually filtered out the second value correctly here.

Comment (szymon-czapracki) [reply lib/tofdb/selftest/src/testcases/tofdb_set_tof_test.c:114]: Is this way ok?

Comment (ncasaril) [reply lib/tofdb/selftest/src/testcases/tofdb_set_tof_test.c:114]: Looks ok. I may need to add the "2m" culling threshold to a mynewt-variable in the future. Then this check would need to be updated of course. 
Thanks.

**Comment by ncasaril** [comment]
@Guilhem23 Can this be merged also?

**Pull request merged**